src/passes/MemoryCopyFillLowering.cpp

```diff
@@ -221,6 +235,15 @@ struct MemoryCopyFillLowering
     }
     module->features.disable(FeatureSet::BulkMemory);
   }
+
+  void VisitTableCopy(TableCopy* curr) {
+    Fatal() << "table.copy instruction found. Memory copy lowering is not "
+      "designed to work on modules with bulk table operations";
+  }
+  void VisitTableFill(TableCopy* curr) {
+    Fatal() << "table.fill instruction found. Memory copy lowering is not "
+    "designed to work on modules with bulk table operations";
+  }
```

Comment:
Same for `table.init` and `memory.init`?

Reply:
The init and drop instructions are only valid when there are passive segments, so it should be sufficient to only check for passive segments.